scormxblock/static/js/src/studio.js

```diff
@@ -117,4 +124,41 @@ function ScormStudioXBlock(runtime, element) {
     runtime.notify('cancel', {});
   });
 
+  $(element).find('input[name=display_type]').bind('click', function() {
```

Comment:
Please comment the js file specially the functions to help understand your implementation to a 3rd developer in the future.